install.md

```diff
@@ -185,7 +211,7 @@ To use the examples in `java-sdk/examples` you can also set the following env:
 
 The following must be setup to use either the Java or Python SDK
 
-    export HARNESS_CA_CERT=/path/to/harness/server/cert.pem
+    export HARNESS_SERVER_CERT_PATH=/path/to/harness/server/cert.pem
     # points to the server's .pem file, used for self-signed
```

Comment:
As a convention I don't descript paths this way anymore. Non-english speakers find this confusing. So I do something like <put-your-path-here> or something that will cause an error if they to not replace it, instead of a "cannot find". I get too many questions about this on mailing lists for things like PIO and the UR.

Reply:
This one and other comments below are about fixes|improvements, but this PR is about the merge. I'll do those fixes in next PR.